File: homework/A1035531_CPE/CPE10480/CPE10480/CPE10480.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main() {
	int a, b;

	while (cin >> a >> b) {
		if ((a == 0) && (b == 0))
			break;
		int count = 0;
		int i = sqrt(a);
		if (i*i < a)
			i = i + 1;
		while (i*i <= b) {
			count++, i++;
		}
		cout << count << endl;
	}
}

File: homework/A1035531_CPE/CPE10480.java
import java.util.*;
public class CPE10480 {
	public static void main(String[] args)
	{
		Scanner in = new Scanner(System.in);
		
		while (in.hasNext())
		{
			int a = in.nextInt();
			int b = in.nextInt();
			int count=0;
		if(a==0&&b==0)
			break;
		for(int i=a;i<=b;i++)	
		{
		if(Math.sqrt(i)%2==0||Math.sqrt(i)%2==1)
			count=count+1;
		}
			System.out.println(count);
		}	
	}
}

